parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -143,6 +143,14 @@ generateAggregatedAPICerts() {
     $AGGREGATED_API_CERTS_SETUP_FILE
 }
 
+configureKubeletServerCert() {
+    KUBELET_SERVER_PRIVATE_KEY_PATH="/etc/kubernetes/certs/kubeletserver.key"
```

Comment:
this will get written for every k8s version even if it only gets used only for k8s 1.15+? Is there a good reason we'd want to only generate the certs for clusters 1.15 and up?

Reply:
the `--tls-cert-file and --tls-private-key-file arguments` should be used in all kubelet versions, I have changed the PR, PTAL.

Reply:
is it supported for all k8s versions?

Reply:
yes, here is the k8s v1.7 parameters: https://github.com/kubernetes/kubernetes/blob/release-1.7/cmd/kubelet/app/options/options.go#L185-L189